docs/source/using/troubleshooting.adoc

```diff
@@ -314,3 +314,28 @@ You can configure the URL by running:
 $ minishift config set check-network-http-host <URL>
 ----
 
+[[github-api-rate-limit-exceeded]]
+== GitHub API rate limit exceeded
+
+When you run `minishift start` or `minishift update`, it might make a request to GitHub to get the latest tarball of the Minishift or OpenShift binary.
+Sometimes, during these requests, you might receive a 403 forbidden status from GitHub if your network reached the API rate limit and the command does not succeed, for example:
+
+----
+Error starting the cluster: Error attempting to download and cache oc: Cannot get the OpenShift
+release version v3.6.0: GET https://api.github.com/repos/openshift/origin/releases/tags/v3.6.0:
+403 API rate limit exceeded for (your IP shows here). (But here's the good news: Authenticated
+requests get a higher rate limit. Check out the documentation for more details.); rate reset in
+17m2.462768522s
+----
+
+GitHub has a rate limiting policy, see link:https://developer.github.com/v3/#rate-limiting[Rate Limiting]. You may have reached this limit for various reasons. For example, your package manager may use GitHub API calls or you are behind a corporate network that makes a lot of GitHub unauthenticated API calls.
+
+Instead of waiting for GitHub to reset the limit for your network so you can run the commands successfully, you can create a personal API token from your GitHub account that determines the rate limit for authenticated requests are higher. See link:https://GitHub.com/blog/1509-personal-api-tokens[Personal API Tokens] for more details on how to generate an API token.
```

Comment:
"Instead of waiting for GitHub to reset the limit for your IP, you can create a link:https://GitHub.com/blog/1509-personal-api-tokens[Personal API Token] from your GitHub account giving you a higher rate limit"